tests/test_build_latex.py

```diff
@@ -12,17 +12,24 @@
 
 import os
 import re
+from itertools import product
 from subprocess import Popen, PIPE
 
 from six import PY3
 
 from sphinx.errors import SphinxError
+from sphinx.util.osutil import cd, ensuredir
 from sphinx.writers.latex import LaTeXTranslator
 
 from util import SkipTest, remove_unicode_literals, with_app, strip_escseq
 from test_build_html import ENV_WARNINGS
 
 
+LATEX_ENGINES = ['pdflatex', 'lualatex', 'xelatex']
+DOCCLASSES = ['howto', 'manual']
+STYLEFILES = ['article.sty', 'fancyhdr.sty', 'fancybox.sty', 'titlesec.sty', 'amsmath.sty',
```

Comment:
I think ``'fancybox.sty'`` is not a dependency of sphinx.sty in master, hence it could be removed from the list here.